packages/web-app/web-app-front/src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
-import App from './app/app';
+import App from './app/App';
```

Comment:
import App from './app/app'.

Reply:
It works on mac and win but it crashes on linux.

Reply:
fixed